dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -273,19 +275,19 @@ CONFIG_AIO=y
 CONFIG_IO_URING=y
 CONFIG_ADVISE_SYSCALLS=y
 CONFIG_MEMBARRIER=y
+CONFIG_KCMP=y
+CONFIG_RSEQ=y
+# CONFIG_DEBUG_RSEQ is not set
```

Comment:
I had impression this is a good one to enable.
cc @melver also relevant to our recent work

Reply:
I don't think it adds much. All it seems to do is: "Terminate the process if a syscall is issued within a restartable sequence." - which looks more useful for debugging the user space process than kernel.